scenarios/Loadfilename_as_column_gcs/README.md

```diff
@@ -0,0 +1,60 @@
+# Workflow: Scenario (Load file name as a column from GCS)
+
+## Scenario
+
+The purpose of this scenario is to capture loaded file name as data from Google Cloud Storage(GCS) on Google Cloud Platform(GCP).
+
+
+*Steps*
+1. Obtain file names and file contents from multiple files in a specific bucket of GCS.
+   the file sample
+   name: files_aaa_20210401
+   contents:
```

Comment:
L10 - L12 lines aren't written in a new line.
So, if you intend to write in other lines, it might need to fix. Would you check it?

Reply:
Thank you for your check! I will add the new lines.